ui/app/models/node.js

```diff
@@ -127,9 +127,9 @@ export default class Node extends Model {
     if (this.isDraining || !this.isEligible) {
       return 'alert-circle-fill';
     } else if (this.status === 'down') {
-      return 'cancel-circle-fill';
+      return 'x-circle-fill';
     } else if (this.status === 'initializing') {
-      return 'node-init-circle-fill';
+      return 'entry-point';
```

Comment:
Using this for initializing nodes now: 
![image](https://github.com/hashicorp/nomad/assets/713991/b5f57c44-de8d-42ca-9c87-a7b8a26869bb)